src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/config/ClientTrackingConfiguration.kt

```diff
@@ -37,10 +37,12 @@ class ClientTrackingInterceptor : HandlerInterceptor {
     if (StringUtils.startsWithIgnoreCase(token, bearer)) {
       try {
         val jwtBody = getClaimsFromJWT(token)
-        val properties = ThreadContext.getRequestTelemetryContext().httpRequestTelemetry.properties
         val user = Optional.ofNullable(jwtBody.getClaim("user_name"))
-        user.map { it.toString() }.ifPresent { properties["username"] = it }
-        properties["clientId"] = jwtBody.getClaim("client_id").toString()
+        user.map { it.toString() }.ifPresent {
```

Comment:
This was updated with https://github.com/ministryofjustice/hmpps-tech-docs/blob/99fff7d837644d573652c7c6f3a3ea78be654eeb/scripts/upgrade-spring-boot-3.bash#L48-L49